src/ResourceManager/Network/Stack/Commands.Network/Models/PSPeering.cs

```diff
@@ -29,7 +29,7 @@ public class PSPeering : PSChildResource
         public int AzureASN { get; set; }
 
         [JsonProperty(Order = 1)]
-        public int PeerASN { get; set; }
+        public uint PeerASN { get; set; }
```

Comment:
Did you intend to also change stack? It runs on an earlier version of Network so it might not be able to take this change.

Reply:
The corresponding change in NRP has already been done.  The NRP API can handle this change.

Reply:
@maddieclayton :  I am not clear about this comment.  What do you mean by the "stack" ? If you mean that changes in azure-sdk/azure-rest-api-specs, then those changes have already been done and merged.

Reply:
If you look at the file path of this file and the file above this one, it is in the "Stack" version of Network, rather than the current stable version of Network.  You likely want to revert these two changes in Stack.